tests/storage/export.go

```diff
@@ -410,12 +417,15 @@ var _ = SIGDescribe("Export", func() {
 		out, stderr, err = exec.ExecuteCommandOnPodWithResults(virtClient, downloadPod, downloadPod.Spec.Containers[0].Name, md5Command(fileAndPathName))
 		Expect(err).ToNot(HaveOccurred(), out, stderr)
 		md5sum := strings.Split(out, " ")[0]
+		Expect(md5sum).To(HaveLen(32))
 		Expect(md5sum).To(Equal(expectedMD5))
 	}
 
 	verifyArchiveGzContent := func(fileName, expectedMD5 string, downloadPod *k8sv1.Pod, volumeMode k8sv1.PersistentVolumeMode) {
 		command := []string{
 			"/usr/bin/tar",
+			"--strip-components",
```

Comment:
Does this make a difference? why do the paths need to change?

Reply:
untar is trying to change permissions on the folder without this.
I am not sure why, but this way I can see it's only attempting to extract the actual disk.img and nothing else